Create documentation files for Site Health Screen in User Handbook (#390)
buddypress/buddypress: docs/user/site-health.md

Some of these Constants are user-definable and will alter the behavior of BuddyPress when the defaults are overridden. This list is made available for your purvey in the event a misdefined Constant may be causing unexpected behavior.

| Constant | Default Value |
| :------- | :------------ |
| BP_VERSION | The installed version of BuddyPress. |
| BP_DB_VERSION | The installed database version of BuddyPress. |
| BP_REQUIRED_PHP_VERSION | The minimum supported PHP version. |
| BP_PLUGIN_DIR | The filesystem directory path (with trailing slash) for the BuddyPress plugin. |
| BP_PLUGIN_URL | The URL directory path (with trailing slash) for the BuddyPress plugin. |
| BP_IGNORE_DEPRECATED | undefined |
| BP_LOAD_DEPRECATED | undefined |
| BP_ROOT_BLOG | 1 (integer) |
| BP_ENABLE_MULTIBLOG | undefined |
| BP_ENABLE_ROOT_PROFILES | undefined |
| BP_DEFAULT_COMPONENT | undefined |

> | BP_XPROFILE_BASE_GROUP_NAME | If **Extended Profiles** component is active this value will be the default avatar setting as selected per the [Settings Discussion](https://wordpress.org/documentation/article/settings-discussion-screen/#avatars) screen. Otherwise, the value is undefined.  |

Author: ```suggestion
| BP_XPROFILE_FULLNAME_FIELD_NAME | If **Extended Profiles** component is active this value will be the primary xprofile field label (see [BuddyPress xProfile fields administration](administration/users/xprofile.md)). Otherwise, the value is undefined. |
```

Changesets [14049](https://buddypress.trac.wordpress.org/changeset/14049) & [14050](https://buddypress.trac.wordpress.org/changeset/14050) changed/corrected the value for `BP_XPROFILE_BASE_GROUP_NAME` which requires this suggested update.
```

Reviewer: @emaralive I believe it should be about the `BP_XPROFILE_BASE_GROUP_NAME` constant not about the `BP_XPROFILE_FULLNAME_FIELD_NAME` one which is documented at line 74.

That is to say: 
```md
| BP_XPROFILE_BASE_GROUP_NAME | If **Extended Profiles** component is active this value will be the label of the primary group of xProfile fields (see [BuddyPress xProfile fields administration](administration/users/xprofile.md)). Otherwise, the value is undefined. |
```

Author: @imath - Yes, you are correct. I had copied and pasted from line 74 and didn't edit any further due to some unknown reason, probably due to a distraction that occurred around the same time.

The goal was to earmark the places where I would make edits and then push another commit.  That stated,  I updated the suggestion and if there is nothing else, I'll updatte .md with the text changes and replace the indicated screenshot with a newer one and then push a commit either later today or tomorrow.

| BP_XPROFILE_FULLNAME_FIELD_NAME | If **Extended Profiles** component is active, this value will be the label of the primary xprofile field (see [BuddyPress xProfile fields administration](administration/users/xprofile.md)). Otherwise, the value is undefined. |
| BP_MESSAGES_AUTOCOMPLETE_ALL | undefined |
| BP_DISABLE_AUTO_GROUP_JOIN | undefined |
| BP_GROUPS_DEFAULT_EXTENSION | undefined |
| BP_MEMBERS_REQUIRED_PASSWORD_STRENGTH | undefined |
| BP_EMBED_DISABLE_PRIVATE_MESSAGES | undefined |
| BP_EMBED_DISABLE_ACTIVITY | undefined |
| BP_EMBED_DISABLE_ACTIVITY_REPLIES | undefined |
| BP_ENABLE_USERNAME_COMPATIBILITY_MODE | undefined |
| BP_AVATAR_DEFAULT_THUMB | undefined |
| BP_AVATAR_DEFAULT | undefined |
| BP_AVATAR_URL | The URL directory path for Avatar uploads. |
| BP_AVATAR_UPLOAD_PATH | The filesystem directory path for Avatar uploads. |
| BP_SHOW_AVATARS | 1 (integer) |

> | BP_AVATAR_ORIGINAL_MAX_WIDTH | 450 (pixels) |

Reviewer: ```suggestion
| BP_AVATAR_ORIGINAL_MAX_WIDTH | 450 (pixels). This constant defines the maximum width of the image BuddyPress will temporary write on the site's server to use as the base image for the Profile photo cropping step. |
```

> | BP_AVATAR_ORIGINAL_MAX_FILESIZE | 5120000 (bits) |

Reviewer: ```suggestion
| BP_AVATAR_ORIGINAL_MAX_FILESIZE | 5120000 (bytes). This constant defines the maximum size (in bytes) of the image BuddyPress will temporary write on the site's server to use as the base image for the Profile photo cropping step. |
```

> | BP_AVATAR_FULL_HEIGHT | 150 (pixels) |

Reviewer: ```suggestion
| BP_AVATAR_FULL_HEIGHT | 150 (pixels). This constant defines the height (in pixels) to use for full size profile photos. |
```

> | BP_AVATAR_FULL_WIDTH | 150 (pixels) |

Reviewer: ```suggestion
| BP_AVATAR_FULL_WIDTH | 150 (pixels). This constant defines the width (in pixels) to use for full size profile photos. |
```

> | BP_AVATAR_THUMB_HEIGHT | 50 (pixels) |

Reviewer: ```suggestion
| BP_AVATAR_THUMB_HEIGHT | 50 (pixels). This constant defines the height (in pixels) to use for small or thumb size profile photos. |
```

> | BP_AVATAR_THUMB_WIDTH | 50 (pixels) |

Reviewer: ```suggestion
| BP_AVATAR_THUMB_WIDTH | 50 (pixels). This constant defines the width (in pixels) to use for small or thumb size profile photos. |
```

> | BP_FORUMS_PARENT_FORUM_ID | 1 (integer) |

Reviewer: ```suggestion
| BP_FORUMS_PARENT_FORUM_ID | 1 (integer). Not used by BuddyPress, left for [bbPress](https://bbpress.org) usage. |
```

> | BP_FORUMS_SLUG | forums (string) |

Reviewer: ```suggestion
| BP_FORUMS_SLUG | forums (string). Not used by BuddyPress, left for [bbPress](https://bbpress.org) usage. |
```

> | BP_SEARCH_SLUG | search (string) |

Reviewer: ```suggestion
| BP_SEARCH_SLUG | search (string). The URL slug to use to perform a BuddyPress global search. Not used internally, but the [BP Search Block add-on](https://wordpress.org/plugins/bp-search-block/) is using it. |
```
